Sources/SyncDataProviders/Bookmarks/BookmarksProvider.swift

```diff
@@ -154,7 +155,7 @@ public final class BookmarksProvider: DataProvider {
         if sent.isEmpty {
             return []
         }
-        let identifiers = sent.compactMap { SyncableBookmarkAdapter(syncable: $0).uuid }
+        let identifiers = sent.compactMap { SyncableBookmarkAdapter(syncable: $0, isPatchRequestPayload: true).uuid }
```

Comment:
SyncableBookmarkAdapter gets a boolean parameter to decide on how the `children` array should be read (because Sync response uses the old format).